code/__DEFINES/dcs/signals.dm

```diff
@@ -264,6 +264,12 @@
 #define COMSIG_MOVABLE_BUCKLE "buckle"
 ///from base of atom/movable/unbuckle_mob(): (mob, force)
 #define COMSIG_MOVABLE_UNBUCKLE "unbuckle"
+///from /atom/movable/proc/buckle_mob, used by the ridable element to see if someone can successfully ride the target
+#define COMSIG_MOVABLE_TRY_MOUNTING "try_mounting"
```

Comment:
```suggestion
#define COMSIG_MOVABLE_BUCKLE_MOB "buckle_mob"
```

Reply:
My concern here is that there's already an existing COMSIG_MOVABLE_BUCKLE fired later in this proc once we're sure we can buckle/mount it. This "signal" is really just an intercept so the ridable element (if one exists) can run its own last minute checks to see if the mob can actually buckle to this atom (you can't ride a skateboard with no legs for example), and stop the buckle if they can't.

Obviously using signals as stand-ins for proc calls isn't great, but there is a distinction between this (a last call for objections to the buckling) and the existing COMSIG_MOVABLE_BUCKLE (which means we've successfully buckled). Would renaming this to COMSIG_MOVABLE_PREBUCKLE work?